schema/CVE_Record_Format.json

```
@@ -301,10 +306,33 @@
                             {
                                 "required": ["version", "status", "versionType", "lessThan"]
                             },
+                            {
+                                "required": ["status", "versionType", "lessThan"]
+                            },
                             {
                                 "required": ["version", "status", "versionType", "lessThanOrEqual"]
-                            }
+                            },
+                            {
+                                "required": ["status", "versionType", "greaterThan"]
+                            },
+                            {
+                                "required": ["status", "versionType", "greaterThanOrEqual"]
+                            },
                         ],
+                        "if": {
+                            "properties": {
+                                "versionType": {"const": "semver-2.0.0"}
```

Since a PATH must be backwards compatible.
```suggestion
                                "versionType": {"const": "semver-2.0"}
```

---

As I said here (https://github.com/CVEProject/cve-schema/pull/371/files#r2210891519) I believe we've resolved this in QWG discussions, determining that "2.0.0" is correct.